### PR TITLE
Update Rule “do-you-use-the-code-health-extensions-in-visual-studio/rule”

### PR DESCRIPTION
**Pull request by vladislav-kir**
Automatically generated by Netlify CMS

### CODE REVIEW

File: rules/do-you-use-the-code-health-extensions-in-visual-studio/rule.md [hunk 1]
```diff
@@ -44,15 +41,19 @@ If you believe the issues being raised are not important, please check the secti
 ### Modify Visual Studio Analysis Ruleset
 
 The goal is to develop a shared ruleset across projects. (Currently this is just the default settings). This will ensure the same standard and quality of code is maintained across all of the company's projects.
-Any project specific rules should be documented in "\_Instructions-CodeHealth.docx" which is to be kept in the solution.
-Please also copy the current version number of this rule into the "\_Instructions-CodeHealth.docx" in order to track what version your existing solution adheres to.
+Any project specific rules should be documented in "_Instructions-CodeHealth.docx" which is to be kept in the solution.
+Please also copy the current version number of this rule into the "_Instructions-CodeHealth.docx" in order to track what version your existing solution adheres to.
```
Comment (bradystroud) [inline rules/do-you-use-the-code-health-extensions-in-visual-studio/rule.md:45]: We typically store docs in Markdown files or project wiki. Not word documents

Check use cases isnt already covered by https://www.ssw.com.au/rules/awesome-documentation/#good-example--the-8-important-documents

File: rules/do-you-use-the-code-health-extensions-in-visual-studio/rule.md [hunk 2]
```diff
@@ -44,15 +41,19 @@ If you believe the issues being raised are not important, please check the secti
...
+![Figure: Add EditorConfig File](add-editor-config.png)
 
-Right Click project &gt; Properties &gt; Code Analysis &gt; Open
+To edit the ruleset, open your .editorconfig file.
 
-![Figure: How to customize rules. By either enabling / disabling rules or packages. Or by modifying the rule severity level](VS-ModifyRules2.png)
+In the Analyzers tab you can change the severity of the rules, for example disable the warning or convert it into an error.
 
-![Figure: How to apply custom ruleset to all projects in a solution](VS-ModifyRules3.png)
+![Figure: Configure your Rules in the in the EditorConfig | Analyzers](configure-rules.png)
```
Comment (bradystroud) [inline rules/do-you-use-the-code-health-extensions-in-visual-studio/rule.md:59]: We have a specific rule for .editorconfig.
https://www.ssw.com.au/rules/consistent-code-style/

I would remove this section and replace it with a link to that rule

File: rules/do-you-use-the-code-health-extensions-in-visual-studio/rule.md [hunk 3]
```diff
@@ -29,13 +27,12 @@ The code quality standard should extend the Visual Studio Analyser. A wide varie
 Search & Install the NuGet packages:
 
 * "Roslyn Security Guard" ([Nuget page for Roslyn Security Guard](https://www.nuget.org/packages/RoslynSecurityGuard/)) - Security audit on .NET Applications.
-* "StyleCop.Analysers" ([Nuget page for StyleCop.Analysers](https://www.nuget.org/packages/StyleCop.Analyzers/1.0.0)) - Ensures C# code style conformity.
-* "tslint" ([Nuget page for tslint](https://www.nuget.org/packages/tslint/)) - Allows for issue tracking of TypeScript projects.
+* (Optional) "StyleCop.Analysers" ([Nuget page for StyleCop.Analysers](https://www.nuget.org/packages/StyleCop.Analyzers/1.0.0)) - Ensures C# code style conformity. It is not compatible with Visual Studio 2022
```
Comment (bradystroud) [inline rules/do-you-use-the-code-health-extensions-in-visual-studio/rule.md:30]: > It is not compatible with Visual Studio 2022

Should we still reccommend this?

**Review by bradystroud — CHANGES_REQUESTED** [review]

File: rules/do-you-use-the-code-health-extensions-in-visual-studio/rule.md (hunk 3 above)
Comment (vladislav-kir) [reply rules/do-you-use-the-code-health-extensions-in-visual-studio/rule.md:30]: I think - no. Can I remove it?

File: rules/do-you-use-the-code-health-extensions-in-visual-studio/rule.md [hunk 4]
```diff
@@ -1,21 +1,19 @@
...
-guid: b0048e8d-b6bb-4af3-9397-a9af6fac1a88
 uri: do-you-use-the-code-health-extensions-in-visual-studio
-created: 2017-03-09T22:11:25.0000000Z
 authors:
-- title: Adam Cogan
-  url: https://ssw.com.au/people/adam-cogan
-- title: Danijel Malik
-  url: https://ssw.com.au/people/danijel-malik
+  - title: Adam Cogan
+    url: https://ssw.com.au/people/adam-cogan
```
Comment (SamBWagner) [inline rules/do-you-use-the-code-health-extensions-in-visual-studio/rule.md:7]: @bradystroud do these spaces break the frontmatter

File: rules/do-you-use-the-code-health-extensions-in-visual-studio/rule.md (hunk 3 above)
Comment (SamBWagner) [reply rules/do-you-use-the-code-health-extensions-in-visual-studio/rule.md:30]: If it's not usable in a modern development setting I'd get rid of it/replace it with something worth mentioning

File: rules/do-you-use-the-code-health-extensions-in-visual-studio/rule.md (hunk 1 above)
Comment (SamBWagner) [reply rules/do-you-use-the-code-health-extensions-in-visual-studio/rule.md:45]: Yeah this seems like a super old and frankly poo recommendation. I'd just replace it with a link to the rule Brady linked

Comment (vladislav-kir) [reply rules/do-you-use-the-code-health-extensions-in-visual-studio/rule.md]: Done - changed this to follow the files from the [https://www.ssw.com.au/rules/awesome-documentation/](https://www.ssw.com.au/rules/awesome-documentation/#good-example--the-8-important-documents), and mentioned the rule itself

File: rules/do-you-use-the-code-health-extensions-in-visual-studio/rule.md (hunk 2 above)
Comment (vladislav-kir) [reply rules/do-you-use-the-code-health-extensions-in-visual-studio/rule.md]: Done - I mention that we can use EditorConfig and then just forward them to the [rule](https://www.ssw.com.au/rules/consistent-code-style/)

File: rules/do-you-use-the-code-health-extensions-in-visual-studio/rule.md (hunk 3 above)
Comment (vladislav-kir) [reply rules/do-you-use-the-code-health-extensions-in-visual-studio/rule.md]: Done - removed recommendation for StyleCop.Analysers. [In this rule](https://www.ssw.com.au/rules/consistent-code-style/) StyleCop  is bad example, so yeah, we shouldn't mention it anymore

File: rules/do-you-use-the-code-health-extensions-in-visual-studio/rule.md (hunk 4 above)
Comment (vladislav-kir) [reply rules/do-you-use-the-code-health-extensions-in-visual-studio/rule.md:7]: @bradystroud, should I return these back to how they were before?

Comment (bradystroud) [reply rules/do-you-use-the-code-health-extensions-in-visual-studio/rule.md:7]: No - they are fine

**Review by bradystroud — APPROVED** [review]
LGTM 🛶

**Pull request merged**